docs/installation_guide.md

```diff
@@ -1,163 +1,238 @@
+
+## Homebrew (optional)
+A package manager that may ease the setup process. This is automatically installed on most MacOS systems, however it requires some setup for Linux. 
+
+Homebrew for Linux:
+
+https://docs.brew.sh/Homebrew-on-Linux
+
+
+Homebrew is not available for Windows, however it is available for WSL. There are other package managers for Windows that might be useful, such as Chocolatey (https://chocolatey.org/), however we have not tested how useful it is here.
+
 
 # Installing
+Now that we have our environment setup, we can start installing the required software to deploy DYNAMOS.
 
-## Helm
-Software to deploy all the DYNAMOS environment
-https://helm.sh/docs/intro/install/
-## Linkerd 
-https://linkerd.io/2.15/getting-started/
 ## kubectl
-CLI tool to interact with Kubernetes: 
+https://kubernetes.io/docs/tasks/tools/
+
+Kubernetes CLI tool. 
+
+Now that docker desktop has installed kubernetes for us, we need to be able to communicate with it, kubectl is a useful CLI tool for this purpose.
+
+Homebrew:
+```bash
+brew install kubectl
+```
 
-On linux you can simply do:
+Snap:
 ```bash
 sudo snap install kubectl --classic
 ```
-https://kubernetes.io/docs/tasks/tools/
-## Homebrew (for linux)
 
-https://docs.brew.sh/Homebrew-on-Linux
+## Helm CLI
+https://helm.sh/docs/intro/install/
 
-## k9s (optional) 
-Preqeqs: Homebrew
-Useful CLI tool to view all currently running kuberentes containers.
-https://k9scli.io/topics/install/
+Tool that is responsible for deploying the microservices to kubernetes based off of helm charts.
 
+Helm can be installed via a script provided by helm or through a package manager
 
-# Secondary Installations
+**Brew**:
+```bash
+brew install helm
+```
+
+**Apt**:
+```bash
+curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+sudo apt-get install apt-transport-https --yes
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+sudo apt-get update
+sudo apt-get install helm
+```
 
 ## Linkerd
-### Prepare cluster
+https://linkerd.io/2.15/getting-started/
+
+**Prereqs**: kubectl
+
+Linkerd is a service mesh for Kubernetes, we use it to install Jaeger (https://www.jaegertracing.io/) a distributed tracing platform onto our cluster.
+
 ```bash
+# Install CLI
+curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install-edge | sh
+
+# Add Linkerd to PATH
+export PATH=$HOME/.linkerd2/bin:$PATH
+
+# Install Linkerd on cluster
 linkerd install --crds | kubectl apply -f -
-linkerd install --set proxyInit.runAsRoot=true | kubectl apply -f -
-linkerd check
+linkerd install | kubectl apply -f -
```

Comment:
This is an incorrect duplicate of the command you give a few lines later, line 126

Reply:
Removed